redfish-core/lib/processor.hpp

```diff
@@ -1110,6 +1110,33 @@ inline void afterGetCpuCoreDataByService(
                         }
                         present = *value;
                     }
+                    else if (propName == "PrettyName")
+                    {
+                        const std::string* prettyName =
+                            std::get_if<std::string>(&propValue);
+                        if (prettyName == nullptr)
+                        {
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        asyncResp->res.jsonValue["Name"] = *prettyName;
+                    }
+                }
+            }
+            else if (interface == "xyz.openbmc_project.Object.Enable")
+            {
+                for (const auto& [propName, propValue] : properties)
+                {
+                    if (propName == "Enabled")
+                    {
+                        const bool* enabled = std::get_if<bool>(&propValue);
+                        if (enabled == nullptr)
+                        {
+                            messages::internalError(asyncResp->res);
+                            return;
+                        }
+                        asyncResp->res.jsonValue["Enabled"] = *enabled;
```

Comment:
Can the loop do break here to skip the other properties?
```
                for (const auto& [propName, propValue] : properties)
                {
...
                        asyncResp->res.jsonValue["Enabled"] = *enabled;
                        break;     <--
                }
```

Reply:
done